### DISCUSSION TITLE
linonetwo/template-list

### DISCUSSION THREAD

**Discussion by giscus** [tiddly-gittly/TiddlyWiki-CPL#50] [Plugins]
# linonetwo/template-list

https://tw-cpl.netlify.app/

**Comment by makiaea** [comment]
looks really helpful, thanks for sharing it!

i'm a little confused: in the readme on github it says: Click on a template will apply //all tags and all fields// to the currentTiddler.

but here it says: Click on a template will apply //the text field// to the currentTiddler.

does it do both?

**Reply by linonetwo** [reply]
Yes, sometimes description on CPL will be outdated. The one on github and plguin's readme.tid are the newest ones.
